Use MacPorts zlib, not a bundled copy, and install dzip to the bin
subdirectory.
--- CMakeLists.txt.orig	2021-04-17 17:22:07.000000000 -0500
+++ CMakeLists.txt	2022-08-21 00:42:14.000000000 -0500
@@ -15,7 +15,6 @@
     target_compile_options(dzip_warnings INTERFACE -Wall -Wextra -Wpedantic)
 endif()
 
-add_subdirectory(external)
 if(MSVC)
     # GUI only supported on Windows with MSVC for now
     add_subdirectory(gui)
@@ -35,5 +34,5 @@
     uncompress.c
     v1code.c
 )
-target_link_libraries(dzip PRIVATE dzip_warnings zlib)
-install(TARGETS dzip DESTINATION .)
+target_link_libraries(dzip PRIVATE dzip_warnings z)
+install(TARGETS dzip DESTINATION bin)
--- dzip.h.orig	2021-04-17 17:22:07.000000000 -0500
+++ dzip.h	2022-08-21 00:41:18.000000000 -0500
@@ -3,7 +3,7 @@
 #include <stdlib.h>
 #include <string.h>
 
-#include "external/zlib/zlib.h"
+#include <zlib.h>
 
 typedef unsigned char uchar;
 
